compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -518,8 +545,18 @@ static LogicalResult setRootConfig(
   SmallVector<int64_t> flowTileSizes =
       getDefaultDistributedLevelTileSizes(linalgOp, minTileSizes, maxTileSizes);
 
+  if (isX86(entryPointFn)) {
+    SmallVector<int64_t> tileSizes = {8, 32, 16};
+    if (numLoops == 3) {
+      return setMatmulPadConfig(entryPointFn, contractionOp, flowTileSizes,
+                                tileSizes, vectorSize);
+    }
+    return setSandboxRootConfig(entryPointFn, contractionOp, flowTileSizes,
+                                tileSizes, vectorSize);
+  }
+
```

Comment:
Would it make sense to also have this configuration for RISC-V? So far I've been trying not to separate both backends so that they both can benefit from all the improvements made. I haven't found strong reasons to separate them either. Unless we want them to diverge due to strategic reasons, I see both targets close enough to share the same pipeline/configuration with different parameters.

Reply:
I'm not sure what's the proper tiling sizes for RISC-V target, so I leave it the same as before. I can update them to use the new pipeline in this PR, but that could regress binary sizes, etc. I don't want to mess up your work. :) Let me know if you'd like me to do the update in this PR.